Generator/Sources/NeedleFramework/Utilities/DependencyGraphPrinter.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  Copyright (c) 2018. Uber Technologies
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+import Foundation
+
+class DependencyGraphPrinter {
+    private let components: [Component]
+    
+    init(components: [Component]) {
+        self.components = components
+    }
+    
+    func printDependencyTree(rootComponentName: String) {
```

Comment:
Let's make the method name more Swifty. How about `printDIStructure(withRootComponentName rootComponentName: String)`